apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -12,21 +9,15 @@ import { ListItemComponent } from '../list-item/list-item.component';
     <div
       class="flex w-fit flex-col gap-3 rounded-md border-2 border-black p-4"
       [class]="customClass">
-      <img
-        *ngIf="type === CardType.TEACHER"
-        src="assets/img/teacher.png"
-        width="200px" />
-      <img
-        *ngIf="type === CardType.STUDENT"
-        src="assets/img/student.webp"
-        width="200px" />
+      <ng-content></ng-content>
 
       <section>
-        <app-list-item
-          *ngFor="let item of list"
-          [name]="item.firstName"
-          [id]="item.id"
-          [type]="type"></app-list-item>
+        @for (item of list; track item.id) {
+          <app-list-item
+            [name]="item[showingProp]"
+            [id]="item.id"
+            (deleteItem)="deleteItem($event)"></app-list-item>
+        }
```

Comment:
this is too specific and not very customizable. If I want to add an icon for only one card, this will be complicated to add.